kitchen.dokken.yml

```diff
@@ -10,6 +10,8 @@ transport:
 provisioner:
   name: dokken
   product_name: chef
+  enforce_idempotency: true
+  multiple_converge: 2
   product_version: <%= ENV['CHEF_VERSION'] || 'latest' %>
   install_strategy: once
```

Comment:
This should be removed as it gets imported from the main kitchen file. However, you should change the driver portion to be the following:
```yaml
driver:
  name: dokken
  # because Docker and SystemD/Upstart
  privileged: true
  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
  env: [CHEF_LICENSE=accept]
```
Note: chef_version.